charts/kuberpult/values.yaml.tpl

```diff
@@ -75,6 +75,9 @@ frontend:
     requests:
       cpu: 500m
       memory: 250Mi
+# Limit for the wait time for resources that support waiting on conditions ( e.g. rollout-status ).
+# This MUST be lower than the timeouts of ALL http proxies in use.
+  waitTimeLimit: 10m
```

Comment:
nitpick:
```suggestion
  maxWaitDurationRollout: 10m
```

Reply:
No sure if the name is actually better. Currently it's really just the rollout endpoint, however this could serve as a limit for every wait endpoint as it's mainly determined by the http request time limit of the proxies.

Reply:
ok, makes sense.
I'd still prefer `maxWaitDuration`. Or maybe `waitTimeout`, as we already have a `networkTimeout`

Reply:
Okay, makes sense